app/notifications/validators.py

```diff
@@ -144,10 +144,19 @@ def check_if_service_can_send_to_number(service, number):
         return international_phone_info
 
 
-def check_content_char_count(template_with_content):
+def check_content_character_count(template_with_content):
     if template_with_content.is_message_too_long():
-        message = f"Text messages cannot be longer than {SMS_CHAR_COUNT_LIMIT} characters. " \
-                  f"Your message is {template_with_content.content_count_without_prefix} characters"
+        message = "Your message is too long."
+        if template_with_content.template_type == SMS_TYPE:
+            message = (
+                f"Text messages cannot be longer than {SMS_CHAR_COUNT_LIMIT} characters. "
+                f"Your message is {template_with_content.content_count_without_prefix} characters."
+            )
+        elif template_with_content.template_type == EMAIL_TYPE:
+            message = (
+                f"Email messages cannot be longer than 7500000 bytes. "
```

Comment:
```suggestion
                f"Emails cannot be longer than 7500000 bytes. "
```

Is 'longer than' the right way to describe something measured in bytes?